packages/salesforcedx-vscode-replay-debugger/syntaxes/apexlog.tmLanguage.json

```diff
@@ -0,0 +1,88 @@
+{
+  "version":
+    "https://github.com/Microsoft/vscode/blob/master/extensions/log/syntaxes/log.tmLanguage.json",
```

Comment:
Could you also add a OSSREADME.md similar to https://github.com/forcedotcom/salesforcedx-vscode/blob/develop/packages/salesforcedx-vscode-visualforce/OSSREADME.json at the top-level of packages/salesforcedx-vscode-replay-debugger?

Basically just link to the actual commit SHA where we took this version from. 

It's good for attribution and also to help us identify which SHA we took this file from.